lib/sw_mapper.rb

```diff
@@ -149,13 +149,11 @@ def positive_int?(str)
 
   # the @id attribute of resource/file elements that match the dor_content_type
   # value is used to tell SearchWorks UI app of specific display needs for objects
-  # The SearchWorks app only wants image ids and needs the corresponding druid
-  # and %2F pre-prended to the file_id
+  # The SearchWorks app only wants image ids which comes from purl_model.sw_image_ids
   # @return [Array<String>] filenames
```

Comment:
It looks like this comment is now (or already was) incorrect?  This looks like it does not return an array (unless `purlxml.sw_image_ids` is a multi-dimensional array).